docs/user/index.rst

```diff
@@ -19,10 +19,7 @@ Read the Docs: documentation simplified
 
    /choosing-a-site
    /integrations
-   /downloadable-documentation
-   /environment-variables
-   /subprojects
-   /localization
+   /explanation/advanced
```

Comment:
Do we need to include the new config file index page here, or is it already included somewhere?

Reply:
It's nested in the `advanced` index :+1:

Reply:
(but the location doesn't align with our general structure)